Packs/QRadar/ReleaseNotes/2_5_21.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### IBM QRadar v3
+
+- Updated to add a sleep to address a problem where integration would wait >60 seconds for a search that was completed.
```

Comment:
Fixed an issue where the integration would wait for over 60 seconds after a search had already completed.